sql/2022/sustainability/stylesheet_count.sql

```diff
@@ -0,0 +1,32 @@
+#standardSQL
+# The percentage of stylesheets used inline v. external
+CREATE TEMPORARY FUNCTION getStylesheets(payload STRING)
+RETURNS STRUCT<remote INT64, inline INT64> LANGUAGE js AS '''
+try {
+  var $ = JSON.parse(payload)
+  var sass = JSON.parse($._sass);
+  return sass.stylesheets;
+} catch (e) {
+  return null;
+}
+''';
+
+SELECT
+  _TABLE_SUFFIX AS client,
+  percentile,
+  COUNTIF(stylesheets.remote = 1) / COUNT(0) AS pct_1_remote,
```

Comment:
If you want this by percentile then you probably mean this:

```suggestion
  APPROX_QUANTILES(SAFE_DIVIDE(stylesheets.remote, (stylesheets.inline + stylesheets.remote)), 1000)[OFFSET(percentile * 10)] AS pct_remote,
```